newcoaster.php

```diff
@@ -0,0 +1,20 @@
+<?php
+require_once('includes/db.php');
+require_once('includes/functions.php');
+
+// Process the data from the form
+
+
+// Could validate data in form from here
```

Comment:
Out in industry, it's usually considered best practice to remove commented out code to keep file sizes down.